Bio/__init__.py

```diff
@@ -84,7 +84,7 @@ class BiopythonDeprecationWarning(BiopythonWarning):
 
     Code marked as deprecated is likely to be removed in a future version
     of Biopython. To avoid removal of this code, please contact the Biopython
-    developers by sending an email to biopython-dev@biopython.org.
+    developers by sending an email mailing list or log an issue on GitHub.
```

Comment:
Isn't this missing a "_to_", or instead say _via the mailing list_ as in other places.

Reply:
Good catch, thanks